Markup/Editor/images/wso2-logo.svg

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 16.0.4, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
This is taken from the wso2 website
http://b.content.wso2.com/sites/all/2017-d7-theme/menu2017/wso2-logo.svg